### ISSUE TITLE
Some Reactome Edges Don't Have CURIE Prefixes on Their Subject and Object

### ISSUE THREAD

**Issue by ericawood** [RTXteam/RTX#1405]
There are many orphan edges in the latest build from Reactome that look like this:
```
         {
             "id": "R-HSA-5244402---REACT:has_input---R-HSA-5244408---identifiers_org_registry:reactome",
             "negated": false,
             "object": "R-HSA-5244408",
             "provided_by": "identifiers_org_registry:reactome",
             "publications": [],
             "publications_info": {},
             "relation": "REACT:has_input",
             "relation_label": "has_input",
             "subject": "R-HSA-5244402",
             "update_date": null
         },
```

Some perpetrating lines:
https://github.com/RTXteam/RTX/blob/f2723816f18d8345cda9213a9a416e395fdf20ba/code/kg2/reactome_mysql_to_kg_json.py#L379-L380
https://github.com/RTXteam/RTX/blob/f2723816f18d8345cda9213a9a416e395fdf20ba/code/kg2/reactome_mysql_to_kg_json.py#L405-L406
https://github.com/RTXteam/RTX/blob/f2723816f18d8345cda9213a9a416e395fdf20ba/code/kg2/reactome_mysql_to_kg_json.py#L436-L437

Thus, there is a lot of good stuff in Reactome that is not present in this build, unfortunately.

**Comment by saramsey** [comment]
No worries, looks like your commit will fix it for 2.6.1!

**Comment by ericawood** [comment]
This is fixed in `KG2.6.1`!
